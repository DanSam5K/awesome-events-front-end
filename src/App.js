import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
import SignUpForm from './component/SignUpForm';
import SignInForm from './component/SignInForm';
import NavBar from './component/NavBar';

function App() {
  return (
    <div className="App container-fluid d-flex justify-content-center flex-column">
      <Router>
        <NavBar className="" />
        <Routes>
          <Route exact path="/" element={<h1>Hello</h1>} />
          <Route exact path="/sign_up" element={<SignUpForm />} />
          <Route exact path="/sign_in" element={<SignInForm />} />
        </Routes>
      </Router>
    </div>
  );
}

export default App;
