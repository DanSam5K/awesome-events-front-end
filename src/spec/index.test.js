import { Provider } from 'react-redux';
import { render } from '@testing-library/react';
import store from '../redux/configureStore';
import App from '../App';

describe('Root page', () => {
  describe('Use Jest snapshots to test the Home page', () => {
    it('renders Home page placeholders as expected', async () => {
      const rootPage = render(
        <Provider store={store}>
          <App />
        </Provider>,
      );

      expect(rootPage.asFragment()).toMatchSnapshot();
    });
  });
});
