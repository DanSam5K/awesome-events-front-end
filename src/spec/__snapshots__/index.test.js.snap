// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root page Use Jest snapshots to test the Home page renders Home page placeholders as expected 1`] = `
<DocumentFragment>
  <div
    class="App container-fluid d-flex justify-content-center flex-column px-0"
  >
    <nav
      class="sticky-top position-absolute navbar navbar-light bg-transparent"
    >
      <div
        class="d-flex justify-between container-fluid"
      >
        <button
          aria-controls="offcanvasNavbar"
          aria-label="Toggle navigation"
          class="border border-4 bg-green shadowed-white navbar-toggler collapsed"
          type="button"
        >
          <span
            class="navbar-toggler-icon"
          />
        </button>
      </div>
    </nav>
    <div
      class="container-fluid d-flex flex-column justify-content-center align-items-center content-height bg-image"
    >
      <img
        bg="transparent"
        class="image-size border-0 mb-3 shadowed"
        src="awesome-events-logo-white.png"
      />
      <h3
        class="light-font"
      >
        <a
          class="light-font"
          href="/sign_in"
        >
          Sign In
        </a>
         or 
        <a
          class="light-font"
          href="/sign_up"
        >
          Sign Up
        </a>
         to access available events and create your own events
      </h3>
    </div>
  </div>
</DocumentFragment>
`;
